wl2/src/css/main.css

```diff
@@ -0,0 +1,22 @@
+.mainContent {
```

Comment:
No .css files should have been generated other than the wl2/src/index.css... The wl2/src/css folder needs to be removed.

Reply:
I deleted the CSS folder and didn't another commit/push that seemed to have automatically been added to my pull request.  Can you check to make sure things are ok and merge if so?  Thanks again.  Also, still confused why I still had that folder.

Reply:
It must've been created when you ran less-watch-compiler previously...